app/Http/routes.php

```diff
@@ -56,3 +56,7 @@
     Route::resource('bios', 'BiosController');
 });
 
+Route::group(['prefix' => 'api'], function () {
+    Route::get('talks/{talkId}', 'Api\TalksController@show');
+    Route::get('user/{userId}/talks', 'Api\UsersController@talks');
+});
```

Comment:
Can do...

`Route::group(['prefix' => 'api', 'namespace' => 'Api'], function () { //...`

...and remove the need for the long namespaces in the actual route definitions.